examples/terra-contract/tools/README.md

```diff
@@ -0,0 +1,113 @@
+# Tools
+This javascript package contains tools for querying and deploying the example contract.
+
+If this is the first time running the code, run the below command to install required packages: 
+
+```
+npm ci
+```
+
+## Query
```

Comment:
A better organization for this document might be two sections: "Testnet Demo" and "Local Development". The first one is quick-and-easy and shows you what the contract does in testnet. The second one walks you through the process of buliding the code, deploying it to a local network, and then running the same query command.

Reply:
Moved to main readme and restructured as you suggested.